core/mutator/mutator.go

```diff
@@ -79,8 +79,8 @@ type ReceiveFunc func([]*QueueMessage) error
 type Receiver interface {
 	// Close stops the receiver and returns only when all callbacks are complete.
 	Close()
-	// Flush sends any waiting queue items.
-	Flush(context.Context)
+	// Flush waits for n items and then sends them.
```

Comment:
This says that Flush waits for n items and then sends them, but from the code for FlushN it looks like it might actually just try sending the number of items and returns an error if there are less than n of them, rather than waiting for n to be in the queue and then sending them?

Reply:
In general, it will wait, but for implementations that can't wait, it will error

Reply:
I'll add a comment.